CONTRIBUTING.md

```diff
@@ -24,6 +24,16 @@ A class should only declare one specifier per specifier type.
 All non-private members need to be documented using Doxygen-style comments.
 For functions and methods, parameters are documented first and return type next.
 
+### Clang
+We use clang-format (v 3.9) to enforce our coding standard.
+To format a specific file, use clang-format -i PATH_TO_FILE/FILENAME when in the /src directory.
```

Comment:
Use `` around the command to make it clear where it begins and ends, like `this`.